docker/rockylinux9/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mipsdkcontainer.azurecr.io/rockylinux9
+
+## Common packages for linux build environment
+RUN yum install -y yum-utils && \
+yum-config-manager --enable crb && \
+yum update -y && \
+yum -y group install "Development Tools" && \
```

Comment:
It looks like a lot of this stuff is already installed on the base image: https://github.com/AzureAD/mip-sdk-for-cpp/pull/3443/files#diff-8d220225500880f9c1256710af99ef181556ced5628fddcb832e92c2b201f47e you don't need to install it again